src/lib/db/DatabaseManager.ts

```diff
@@ -9,84 +9,159 @@ import { v4 as uuidv4 } from "uuid";
-  },
-  {
-    id: uuidv4(),
-    name: "Watch TV",
-    categories: ["Distract"],
-    checked: false,
-    infoUrl: "https://google.com/tv",
-    imageUrl:
-      "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
-    timestamp: new Date().toISOString(),
-  },
-  {
-    id: uuidv4(),
-    name: "Call a friend",
-    categories: ["Distract", "Change status"],
-    checked: false,
-    infoUrl: "https://example.com/call",
-    imageUrl:
-      "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
-    timestamp: new Date().toISOString(),
-  },
-  {
-    id: uuidv4(),
-    name: "See a friend",
-    categories: ["Distract", "Change status"],
-    checked: false,
-    infoUrl: "https://example.com/call",
-    imageUrl:
-      "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
-    timestamp: new Date().toISOString(),
-  },
-];
+const seedData = {
+  categories: [
+    {
+      id: uuidv4(),
+      name: "Replace",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "Barrier",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "Distract",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "Change Status",
+      timestamp: new Date().toISOString(),
+    },
+  ],
+  toolkit: [
+    {
+      id: uuidv4(),
+      name: "Listen to my favourite music",
+      categories: ["Replace", "Barrier"],
+      checked: false,
+      infoUrl: "https://google.com/music",
+      imageUrl:
+        "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "Watch TV",
+      categories: ["Distract"],
+      checked: false,
+      infoUrl: "https://google.com/tv",
+      imageUrl:
+        "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "Call a friend",
+      categories: ["Distract", "Change status"],
+      checked: false,
+      infoUrl: "https://example.com/call",
+      imageUrl:
+        "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
+      timestamp: new Date().toISOString(),
+    },
+    {
+      id: uuidv4(),
+      name: "See a friend",
+      categories: ["Distract", "Change status"],
+      checked: false,
+      infoUrl: "https://example.com/call",
+      imageUrl:
+        "https://daily.jstor.org/wp-content/uploads/2023/01/good_times_with_bad_music_1050x700.jpg",
+      timestamp: new Date().toISOString(),
+    },
+  ],
+};
 
 class DatabaseManager {
   dbInstance: RxDatabase | null = null;
 
   async initialiseDatabase() {
+    console.groupCollapsed("Initialising Database");
```

Comment:
This is logging every time we access the database - could we change the phrasing to "querying database" or insert into the if statement to avoid confusion

Reply:
Yeah sure. I'm not on my computer so can't edit myself: can you make the edit suggestion and I'll accept?

I suggest:

1. Changing the log to "Accessing Database"
2. In a future branch, changing the method's name to `accessDatabase()`

Reply:
```suggestion
    console.groupCollapsed("Accessing Database");
```